doc/misc/undocumented-preferences.md

```diff
@@ -0,0 +1,20 @@
+# Why the following scanner preferences are undocumented
+
+Some scanner preferences are found in the scanner code but are not documented in the man page. This is because such preferences are for internal usage or just preferences to be set via the client, and there is not intention to be set via the configuration file. The following are the preferences and what are they used for, and the reason why are not documented in the man page.
+
+- auth_port_ssh: This setting is scanner wide. Not to be set in the config file since it has priority. This should be set by the client. Internally used. It will be set by the client, else it is checked in the kb, else defaults to 22 (SSH default port).
+- config_file: It is set via command line. If set in the configuration file, the scanner loads configuration set in the file where this setting is pointing to.
```

Comment:
```suggestion
- config_file: Location of the config file. It can only be set vie the command line.
```
This one is actually included within the man page